int8 number
bool high
int64 duration_ms
---
bool result
